apps/elixir_ls_utils/test/support/example_config_file.jsonc

```diff
@@ -0,0 +1,7 @@
+# This is the configuration file for ElixirLS
```

Comment:
Comment should be `//` as @lukaszsamson  comment above.

Reply:
Good catch, fixed!